### ISSUE TITLE
Sensitive Data

### ISSUE THREAD

**Issue by renatocava** [lecy/Import-Data-Into-R#3]
Hi @lecy , thanks for your repo. My question is: If I share the code on github, so the code show the public link to the data, ¿it's not the same?. I mean ¿anyone with that link could acces the data?

Greetings

**Comment by lecy** [comment]
Yes, if you upload data to regular public GitHub repositories it is accessible to anyone. 

If you are working with sensitive non-public data you would need to store the data on a secure server and you would need to add a credential layer so that you could limit who could access the raw data. 

Alternatively, you can use private GitHub repos for internal collaborations with teams. The dataset in the repo would at least not be published.
